src/plugins/inputs/part.js

```diff
@@ -9,26 +9,36 @@ exports.commands = ["close", "leave", "part"];
 exports.allowDisconnected = true;
 
 exports.input = function(network, chan, cmd, args) {
-	if (chan.type === Chan.Type.LOBBY) {
+	const target = args.length === 0 ? chan : _.find(network.channels, {name: args[0]});
+
+	if (typeof target === "undefined") {
+		chan.pushMessage(this, new Msg({
+			type: Msg.Type.ERROR,
+			text: "Cannot part channel " + target.name + ": Channel not found."
```

Comment:
`target.name` is undefined here.

Reply:
Ah yeah I remember now why I did not go for the ternary at first..